### PR TITLE
Italian Translation File - v1

### PR DESCRIPTION
**Pull request by aamodio**
This is v1 of Italian Translation file.

### CODE REVIEW

File: viewer/src/locales/Italian.js [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    "You can improve the translation here":
+        "Migliora la traduzione qui",
+
+    // Initial screen
+    "#helpButton - learn more about the app":
+        "#helpButton - Istruzioni all'uso dell'app",
+    "#optionsButton - see the available options":
+        "#optionsButton - Opzioni disponibili",
+    "powered with":
+        "Powered with",
```
Comment (RussCoder) [inline viewer/src/locales/Italian.js:35]: Is it intentional? Or you just forgot about this line?

Comment (aamodio) [reply viewer/src/locales/Italian.js:35]: Intentional, is cooler said in English :-)

**Comment by RussCoder** [comment]
Well done!

**Comment by r-pankevicius** [comment]
Il Canto degli Italiani!

**Comment by aamodio** [comment]
Not as in Russia or in Baltic Republics but DjVu was quite popular in Italy.
Nessun dorma! There is a new efficient way to serve DjVu files to people.

**Comment by r-pankevicius** [comment]
@aamodio 

I was only excited that this project grows if we see the translations; more languages every few months are added. When I joined (for some reasons I had to evaluate possible alternatives and I've chosen it as the best at that moment) it was the one of few. But I put my bets on it and I see it's going forward quite well.

**Pull request merged**